test/lib/ufe/testRename.py

```diff
@@ -420,14 +420,10 @@ def testRenameNotifications(self):
         ufeObs = TestObserver()
 
         # We start off with no observers
-        self.assertFalse(ufe.Scene.hasObjectAddObserver(ufeObs))
-        self.assertFalse(ufe.Scene.hasObjectDeleteObserver(ufeObs))
-        self.assertFalse(ufe.Scene.hasObjectPathChangeObserver(ufeObs))
-
-        # Add the UFE observers we want to test
-        ufe.Scene.addObjectAddObserver(ufeObs)
-        ufe.Scene.addObjectDeleteObserver(ufeObs)
-        ufe.Scene.addObjectPathChangeObserver(ufeObs)
```

Comment:
Again you need to keep the old test code around along with your new code and use an if for UFE preview version.

Reply:
addressed f4f6bd4a848585dfb332e3ee75ac98923af250b5

Also fixed some additional unit tests that were failing:  **testContextOps** , **testDeleteCmd**